spec/sidekiq_publisher/datadog_apm_spec.rb

```diff
@@ -180,7 +183,7 @@ def write(trace)
 
     match do |span|
       self.error_type = span.get_tag("error.type")
-      self.error_msg = span.get_tag("error.msg")
+      self.error_msg = span.get_tag("error.message").presence
```

Comment:
We can remove the `presence` call now that we aren't falling back to `msg` anymore.